charter.html

```diff
@@ -309,16 +309,15 @@ <h3>
             </dd>
           </dl>
         </section>
-        <section id="tentative">
+        <section id="incubation">
           <h3>
-            Tentative Specifications
+            Tentative Deliverables
           </h3>
           <p>
             Depending on the <a href=
             "https://www.w3.org/community/webmachinelearning/">Community
-            Group</a> progress, this Working Group may deliver the following
-            W3C normative specifications after sufficient incubation in the
-            Community Group:
+            Group</a> incubation progress,  interest from multiple implementers, and the consensus
```

Comment:
```suggestion
            Group</a> incubation progress, interest from multiple implementers, and the consensus
```
Nit: remove extra whitespace.